docs/content/guide/component.ngdoc

```diff
@@ -282,7 +282,7 @@ it upwards to the heroList component, which updates the original data.
 </file>
 <file name="editableField.html">
   <span ng-switch="$ctrl.editMode">
-    <input ng-switch-when="true" type="text" ng-model="$ctrl.fieldValue">
+    <input ng-switch-when="true" type="{{::$ctrl.fieldType}}" ng-model="$ctrl.fieldValue">
```

Comment:
I'd remove the `::` (so we don't introduce another concept and because there is nothing preventing the `fieldType` to change dynamically :smiley:)